web/packages/teleport/src/services/resources/types.ts

```diff
@@ -63,6 +63,29 @@ type RoleConditions = {
   node_labels?: Record<string, string[]>;
 };
 
+export type RoleOptions = {
```

Comment:
Add a JSDoc for this? Also since this is using `snake_case`,  mention that the naming is exact representation as object available in the backend.